[2021-08-10 18:59:39,166] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask2 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 18:59:39,177] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask2 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 18:59:39,178] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:59:39,178] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-08-10 18:59:39,178] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:59:39,248] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): Covidtask2> on 2021-08-10T13:14:01.532369+00:00
[2021-08-10 18:59:39,254] {standard_task_runner.py:52} INFO - Started process 35321 to run task
[2021-08-10 18:59:39,266] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Covid', 'Covidtask2', '2021-08-10T13:14:01.532369+00:00', '--job-id', '358', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpftkixu6l', '--error-file', '/tmp/tmpg858lj98']
[2021-08-10 18:59:39,269] {standard_task_runner.py:77} INFO - Job 358: Subtask Covidtask2
[2021-08-10 18:59:39,427] {logging_mixin.py:104} INFO - Running <TaskInstance: Covid.Covidtask2 2021-08-10T13:14:01.532369+00:00 [running]> on host deepika-Inspiron-5570
[2021-08-10 18:59:39,690] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Covid
AIRFLOW_CTX_TASK_ID=Covidtask2
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T13:14:01.532369+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T13:14:01.532369+00:00
[2021-08-10 18:59:47,588] {python.py:151} INFO - Done. Returned value was: None
[2021-08-10 18:59:47,590] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Covid, task_id=Covidtask2, execution_date=20210810T131401, start_date=20210810T131439, end_date=20210810T131447
[2021-08-10 18:59:47,754] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-10 18:59:47,768] {local_task_job.py:151} INFO - Task exited with return code 0
